lexers/testdata/rust.actual

```diff
@@ -15,9 +15,11 @@ fn main() {
     let r#type = "valid";
     let i: i128 = 1117;
 
-    let s = "March
```

Comment:
Thanks for doing this! 

The old version of this should still work I think, according to the issue? ie.

```rust
let s = "March
April"
```

Reply:
No problem.

Yes, and it still does, I just added `\n` for testing, but `\n` is in the last string so maybe it's redundant, Should I remove the `\n`?

Reply:
Cool, just checking!